### DISCUSSION TITLE
1 workflow awaiting approval

### DISCUSSION THREAD

**Discussion by TrustBetOC** [community/community#77758] [New to GitHub]
Hello everyone!
I have a question I am hoping someone may be able to provide clarity on. I have forked the Solana governance UI to set up a DAO through Realms and uploaded my code for the required personalisation of my DAO. I have no issues but just still cant move forward from a preview of my work on Realms until my GitHub workflow is approved by a maintainer. Is there a step I am missing because its been a week and I think in another week it will just automatically close the pull request if not approved.

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Reply by TrustBetOC** [reply]
Thank you very much for your help, I will have a look.
